test/rules/tags-check.spec.js

```diff
@@ -14,46 +14,46 @@ ruleOptions[ruldId] = {
 
 describe('Rules: ' + ruldId, () => {
   it('Tag <a> should have requered attrs [title, href]', () => {
-    var code = '<a>blabla</a>'
-    var messages = HTMLHint.verify(code, ruleOptions)
+    const code = '<a>blabla</a>'
+    const messages = HTMLHint.verify(code, ruleOptions)
     expect(messages.length).to.be(2)
     expect(messages[0].rule.id).to.be(ruldId)
     expect(messages[1].rule.id).to.be(ruldId)
   })
   it('Tag <a> should not be selfclosing', () => {
-    var code = '<a href="bbb" title="aaa"/>'
-    var messages = HTMLHint.verify(code, ruleOptions)
+    const code = '<a href="bbb" title="aaa"/>'
+    const messages = HTMLHint.verify(code, ruleOptions)
     expect(messages.length).to.be(1)
     expect(messages[0].rule.id).to.be(ruldId)
   })
   it('Tag <img> should be selfclosing', () => {
-    var code = '<img src="bbb" title="aaa" alt="asd"></img>'
-    var messages = HTMLHint.verify(code, ruleOptions)
+    const code = '<img src="bbb" title="aaa" alt="asd"></img>'
+    const messages = HTMLHint.verify(code, ruleOptions)
     expect(messages.length).to.be(1)
     expect(messages[0].rule.id).to.be(ruldId)
   })
   it('Should check optional attributes', () => {
-    var code = '<script src="aaa" async="sad" />'
-    var messages = HTMLHint.verify(code, ruleOptions)
+    const code = '<script src="aaa" async="sad" />'
+    const messages = HTMLHint.verify(code, ruleOptions)
     expect(messages.length).to.be(1)
     expect(messages[0].rule.id).to.be(ruldId)
   })
   it('Should check redunant attributes', () => {
-    var code = '<main role="main" />'
-    var messages = HTMLHint.verify(code, ruleOptions)
+    const code = '<main role="main" />'
+    const messages = HTMLHint.verify(code, ruleOptions)
     expect(messages.length).to.be(2)
     expect(messages[0].rule.id).to.be(ruldId)
     expect(messages[1].rule.id).to.be(ruldId)
   })
   it('Should be extendable trought config', () => {
-    var code = '<sometag></sometag>'
-    var messages = HTMLHint.verify(code, ruleOptions)
+    const code = '<sometag></sometag>'
+    const messages = HTMLHint.verify(code, ruleOptions)
     expect(messages.length).to.be(2)
     expect(messages[0].rule.id).to.be(ruldId)
   })
   it('Should check required attributes with specifyed values', () => {
-    var code = '<sometag attrname="attrvalue" />'
-    var messages = HTMLHint.verify(code, ruleOptions)
+    let code = '<sometag attrname="attrvalue" />'
```

Comment:
why let?

Reply:
Because it gets reassigned 3 lines below